ecowitt2mqtt/hass.py

```diff
@@ -134,6 +140,20 @@ class EntityDescription:
         device_class=DEVICE_CLASS_TEMPERATURE,
         unit_class=UNIT_CLASS_TEMPERATURE,
     ),
+    DATA_POINT_LIGHTNING: EntityDescription(
+        platform=PLATFORM_SENSOR,
+        icon="mdi:weather-lightning",
+        unit=UNIT_STRIKES,
+    ),
+    DATA_POINT_LIGHTNING_NUM: EntityDescription(
+        platform=PLATFORM_SENSOR,
+        icon="mdi:map-marker-distance",
+        unit=UNIT_KM,
+    ),
+    DATA_POINT_LIGHTNING_TIME: EntityDescription(
```

Comment:
So currently Home Assistant 2021.12.10 does not like value for "lightning_time" as unix epoch and the value reports as "Unknown". Not sure if we need to convert the value to ISO8601 for this to be displayed properly. I have not seen many sensors report a datetime myself so still investigating this.

Reply:
I tested sending Home Assistant a string with the date time in ISO8601. Home Assistant correctly displays a value indicating the number of days since the date time. In my test case Home Assistant showed a value of "3 days ago".

```python3
import datetime
ecowitt_value = 1642973158
ha_value = datetime.datetime.utcfromtimestamp(ecowitt_value).replace(tzinfo=datetime.timezone.utc).isoformat()
print(ha_value)
```

```
2022-01-23T21:25:58+00:00
```

Reply:
To clarify, I'm wanting to see how the raw data from the Ecowitt device looks. Is that `1642973158`?

Reply:
Yes, that is correct the raw value is `lightning_time=1642973158`.